src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/wrapper/PKCS11.java

```diff
@@ -220,14 +231,15 @@ private native CK_VERSION connect(String pkcs11ModulePath,
      * Disconnects the PKCS#11 library from this object. After calling this
      * method, this object is no longer connected to a native PKCS#11 module
      * and any subsequent calls to C_ methods will fail. This method is for
-     * internal use only.
+     * internal use only.  Please don't use this method other than finalization.
+     *
```

Comment:
Perhaps refer to the code in which the cleanup occurs.

Reply:
Good point!  A reference to the releaserFor() method is added.